protos/Readme.md

```diff
@@ -5,12 +5,10 @@
 ``` brew install swift-protobuf```
 3. If you don't have dart sdk on your computer please install it.
 ```brew isntall dart```
-4. Get the latest dart-protoc-plugin from https://github.com/dart-lang/dart-protoc-plugin
-5. Go to dart-protoc-plugin-<<version>> folder and call
-```pub install```
-6. Add plugin path to PATH
-7. Make sure protoc + protoc-gen-dart + dart bins are all in the same path
-8. Run the following command from the protos folder
+4. Run ```pub global activate protoc_plugin```
+5. OPTIONAL Add plugin path to PATH
+6. Make sure protoc + protoc-gen-dart + dart bins are all in the same path
```

Comment:
I'd suggest using commas instead of pluses. Also, please reword the ending. What means "in the same path"? I'm pretty sure it's not required to have those tools in the same folder.

Reply:
this is not needed left over from the old times removed it :)